_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_vaapi.cpp

```diff
@@ -280,16 +280,17 @@ namespace MfxHwVP9Encode
         Zero(segPar);
 
         mfxFrameInfo const & dstFi = task.m_pSegmentMap->pSurface->Info;
-        mfxU32 dstW = dstFi.Width;
+        // driver seg map is always in 16x16 blocks because of HW limitation
+        constexpr mfxU16 dstBlockSize = 16;
```

Comment:
I'd rather name this one ```blockSize```

Reply:
There are 2 different buffer sizes here - srcBlockSize(set by user) and dstBlockSize(to send to driver)